plugins/se/se.cpp

```diff
@@ -168,18 +180,31 @@ static int fetch(float *avatarPos, float *avatarFront, float *avatarTop, float *
 }
 
 static bool tryInit(const std::multimap< std::wstring, unsigned long long int > &pids) {
+	const std::vector< std::string > names{
 #ifdef OS_LINUX
-	if (initialize(pids, L"hl2_linux")) {
-		return true;
-	}
+		"hl2_linux",
 #endif
+		"left4dead2.exe", "left4dead.exe"
+	};
 
-	if (initialize(pids, L"left4dead2.exe")) {
-		return true;
-	}
-
-	if (initialize(pids, L"left4dead.exe")) {
-		return true;
+	for (const auto &name : names) {
+		const auto id = Process::find(name, pids);
+		if (!id) {
+			continue;
+		}
+#ifdef OS_WINDOWS
+		proc.reset(new ProcessWindows(id, name));
+#else
+		isWin32 = Process::isWine(id);
```

Comment:
```suggestion
		isWin32 = HostLinux::isWine(id);
```
Imo this makes it more clear that this is a function that simply is not available on Windows (as in that case `Process` is inheriting from `HostWindows`.

Reply:
I agree.